ext/ouroboros_api/main.php

```diff
@@ -151,10 +151,7 @@ public function __construct(array $post)
             );
         }
         if (array_key_exists('description', $post)) {
-            $this->description = filter_var_ex(
-                $post['description'],
-                FILTER_SANITIZE_STRING
-            );
+            $this->description = $post['description'];
```

Comment:
this feels like it _would_ be a security concern, if anything was reading `description` and expecting it to be safe -- but as far as I can see nothing is reading `description`? (and if at some point in the future we want to read `description`, it should be sanitised-on-output like everything else)